apps/frontend/assets/src/adminPanelApp/globalStyle.css

```diff
@@ -37,7 +37,7 @@ body {
   top: 0;
   right: 0;
   bottom: 0;
-  overflow: hidden;
+  overflow-y: scroll;
```

Comment:
the body overflow should always be hidden, the overflow should be handle on modal side.